com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -677,39 +976,67 @@ struct ScreenSpaceShadowData
 
         static MaterialPropertyBlock m_LightLoopDebugMaterialProperties = new MaterialPropertyBlock();
 
-        bool HasLightToCull()
+        static Vector4 GetZBinBufferEncodingParams(HDCamera hdCamera)
         {
-            return m_TotalLightCount > 0;
+            // encodingParams = { n, log2(f/n), 1/n, 1/log2(f/n) }
+            // n = 0.1. See ComputeFixedPointLogDepth.
+            float n = 0.1f;
```

Comment:
why not using camera.nearClipPlane ?

Reply:
1. It could be oblique, which means that min-z depends on the pixel coordinate.
2. Small values of the near plane are costly in terms of precision and we only have 8192 bins, so 13 bits of precision.